### DISCUSSION TITLE
Türkçe Dili

### DISCUSSION THREAD

**Discussion by tirtifil81** [IBAX-io/go-ibax#565] [Bug Submission]
Points are not displayed in Turkish Language.It appears when translated into English.

**Comment by ibaxio** [comment]
Try to use Chrome.

Translator bots is not accurate enough for specialized technical words.

The IBAX team is recruiting volunteers to translate the official website. Due to the rapid growth of the Turkish community, Turkish will be uploaded in the next revision.

We also welcome translations from community members in other languages.

Unleash the power of our community and make us stronger.
